veps/sig-compute/vm_video_configuration_field.md

```diff
@@ -0,0 +1,118 @@
+# VEP #35: Add a Video configuration field for VMs
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+# Overview
+We want to allow VM-owners to explicitly set the video device type when needed.
+
+## Motivation
+In KubeVirt, the default video device type for AMD64 architectures is VGA for BIOS-based VMs and Bochs for EFI-based VMs.
+For Arm and s390x architectures, we have transitioned to using the virtio video device.
+The use of VGA/Bochs can impose limitations on video functionality, such as restricted display resolution settings, particularly for Windows virtual machines.
+However, maintaining compatibility with older guest operating systems (e.g., RHEL 5/6) that rely on VGA/Bochs remains important.
+To balance modernization with backward compatibility, KubeVirt continues to use VGA for legacy BIOS-based workloads.
+
+## Goals
+Ensuring VM-owners won't have a limited resolution experience with their VMs
+
+## Definition of Users
+VM owners who require specific video resolutions that are unavailable when using VGA or Bochs as the video device.
+
+## User Stories
+As an AMD64 VM owner, I want my VM to provide high-resolution display settings when accessed via RDP, VNC,
+or other remote access methods, so that I have a better user experience when working with graphical applications.
+
+## Repos
+- [KubeVirt](https://github.com/kubevirt/kubevirt)
+- [common-templates](https://github.com/kubevirt/common-templates)
+- [common-instancetypes](https://github.com/kubevirt/common-instancetypes)
+
+
+## Design
+
+### Option 1: Default to Virtio Instead of VGA/Bochs
+#### Description
+Instead of relying on VGA/Bochs as the default video device for AMD64 architectures, 
+KubeVirt would transition to virtio as the new default across all architectures.
+This change would provide better video performance, higher resolutions,
+and improved compatibility with modern operating systems,especially Windows VMs.
+
+**Pros:**
+- **Reduced Configuration:** Users won't need to manually switch to virtio for better video performance.
+
+**Cons:**
+- **Backward Compatibility:** we might need to adjust a fall-back logic to maintain full backward-computability
```

Comment:
This is a very valid point, and we should think twice about it.
I noticed some difference in bochs to virtio-vga, e.g. 8mb vs 16mb mem, the way the default resolution form OVMF is applied. But I did not notice anything what worries me, but it does not mean that there is no issue I did not recognize.